### ISSUE TITLE
Add DirecTV Stream to MSO Providers

### ISSUE THREAD

**Issue by thegpfury** [yt-dlp/yt-dlp#1953]
### Checklist

- [X] I'm reporting a feature request
- [X] I've verified that I'm running yt-dlp version **2021.12.01**. ([update instructions](https://github.com/yt-dlp/yt-dlp#update))
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues including closed ones. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Description

Can DirecTV Stream be added to the MSO providers? 

Looks like it uses the standard ATT login account.

**Comment by pukkandan** [comment]
DTV is already supported

**Comment by thegpfury** [comment]
DirecTV Stream is a separate login as best as I can tell, does not use the same DirecTV signin.

**Comment by pukkandan** [comment]
Did not know that 😓

Can you test with `ATTOTT` and `DTV` just to make sure it's not either of these

**Comment by thegpfury** [comment]
Yup, tried both of those, ATT signs in with normal ATT creds and fails with channel not avail since TV isnt thru them, and ATTOTT doesnt work either.

**Comment by pukkandan** [comment]
Sorry, forgot to re-open

**Comment by nobodyspecial** [comment]
Any progress on this?
